### PR TITLE
Fix various pkgcheck warnings in openjdk

### PR DESCRIPTION
**Pull request by floppym**

### CODE REVIEW

File: dev-java/openjdk/openjdk-8.332_p09.ebuild
Comment (gyakovlev) [inline dev-java/openjdk/openjdk-8.332_p09.ebuild:208]: oops, looks like I did it for 11, but not for the rest.
thanks for catching.

File: dev-java/openjdk-jre-bin/openjdk-jre-bin-17.0.3_p7.ebuild
Comment (gyakovlev) [inline dev-java/openjdk-jre-bin/openjdk-jre-bin-17.0.3_p7.ebuild:75]: Honestly I prefer this to use unique files, despite those being identical.

**Review by gyakovlev — COMMENTED** [review]
Thanks!
Let me test couple of bootstrap scenarios before merging.
the main problem in choosing bootstrap vm was that we have to support pre-gentoo-vm and post-gentoo-vm installations as an upgrade path and there were some tricky scenarios.

I'm not sure about `EPREFIX -> BROOT` at this point, as eclasses still use `DEPEND` to determine  vm, while they work on `EAPI7+`, they are not really cross friendly really. but this does not hurt either, so let us change it to `BROOT` indeed.

File: dev-java/openjdk-jre-bin/openjdk-jre-bin-17.0.3_p7.ebuild
Comment (floppym) [reply dev-java/openjdk-jre-bin/openjdk-jre-bin-17.0.3_p7.ebuild:75]: This causes pkgcheck to emit a warning. We can drop the commit when rebasing if you really don't care about the warning.

Comment (gyakovlev) [reply dev-java/openjdk-jre-bin/openjdk-jre-bin-17.0.3_p7.ebuild:75]: currently it does not emit this due to different  copyright year.
but does emit it for `openjdk-jre-bin`

I don't really care about this warning, but if we going to dedupe this file I'd prefer to remove version specifier altogether, so  it's clear that it's version-independent file installed with slot appended. just to avoid possible confusion in the future in case we need to make version specific modifications.

**Comment by floppym** [comment]
I leave it to the maintainers as to whether to drop the env file related commits when merging.

**Review by Flowdalic — APPROVED** [review]

**Comment by gyakovlev** [comment]
thanks!
tested and merged.

**Pull request closed without merging**
